crates/sui-graphql-rpc/src/types/move_object.rs

```diff
@@ -27,6 +27,7 @@ pub(crate) struct MoveObject {
 
 pub(crate) struct MoveObjectDowncastError;
 
+/// The representation of an object as a Move Object.
```

Comment:
Since this shows up on graphql schema, maybe a little bit on why someone would want to view an object as a Move object? See its contents, module that governs it, etc.